node/src/main/scala/coop/rchain/node/NodeEnvironment.scala

```diff
@@ -20,24 +20,24 @@ object NodeEnvironment {
 
   class InitializationException(msg: String) extends RuntimeException
 
-  def create(conf: Configuration)(implicit log: Log[Task]): Effect[NodeIdentifier] =
+  def create(conf: Configuration)(implicit log: Log[Task]): Task[NodeIdentifier] =
     for {
-      dataDir <- Task.delay(conf.server.dataDir.toFile).toEffect
+      dataDir <- Task.delay(conf.server.dataDir.toFile)
```

Comment:
I'd remove the `.toEffect` calls, but maybe leave the `Effect` type alias and its usage? Just in case we'd want to migrate to ZIO or sth...

OTOH ZIO has a Task type of the same shape, so just the imports would change.
If we were to migrate to a bifunctor, there's no big difference between `Task[A] -> ZIO[E, A]` and `Effect[A] -> Effect[E, A]`

WDYT? When's a type alias worth it?

Reply:
see my reply to @lukasz-golebiewski

Reply:
If we do things right chainging the `IO` type should be matter of modifing module `node` - but prooject is not that state yet (see `comm` queue machanism, see casper tests).

Global rename `s/monix.Task/zio.Task` will be least of our concerns

Reply:
FWIW it's been quite easy to port one of the tests executing end-to-end through rholang :)
https://github.com/rchain/rchain/compare/dev...ArturGajowy:try_zio